web/packages/teleport/src/Navigation/Navigation.tsx

```diff
@@ -68,6 +68,26 @@ export function getFirstRouteForCategory(
   );
 }
 
+function getFeatureForRoute(
+  features: TeleportFeature[],
+  route: history.Location<unknown> | Location
+) {
+  const feature = features
+    .filter(feature => Boolean(feature.route))
+    .find(feature =>
+      matchPath(route.pathname, {
+        path: feature.route.path,
+        exact: feature.route.exact,
+      })
+    );
+
+  if (!feature) {
+    return;
+  }
+
+  return feature;
+}
```

Comment:
Nit, but this would work as well I think:

```typescript
function getFeatureForRoute(
  features: TeleportFeature[],
  route: history.Location<unknown> | Location
) {
  return features
    .find(feature =>
      feature.route && matchPath(route.pathname, {
        path: feature.route.path,
        exact: feature.route.exact,
      })
    );
}
```